ls1intum/Artemis
# `Programming exercise`: Fill title and short name during import

### Checklist
#### General



#### Client


### Motivation and Context
Stephan requested that we should also import title and shortname for programming exercises.

### Description
This PR removes the cleanup of title and shortname during import of programming exercises.

### Steps for Testing

Prerequisites:
- 1 Instructor


1. Log in to Artemis
2. Navigate to Exercise Administration
3. Click "Import Programming Exercise"
4. Chose an existing Programming Exercise
5. Verify that title and short-name are imported as well (screenshot)


### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test9)

### Review Progress







#### Code Review
#### Manual Tests


### Test Coverage







### Screenshots
![image](https://github.com/ls1intum/Artemis/assets/78978542/ee5439a1-f079-451b-9a65-d8c13563b5cb)
